.gitlab-ci.yml

```diff
@@ -168,16 +168,16 @@ variables:
   # Build images versions
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
-  DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v41089396-d65d75ec
-  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v41089396-d65d75ec
-  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v41089396-d65d75ec
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v41089396-d65d75ec
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v41089396-d65d75ec
+  DATADOG_AGENT_BUILDIMAGES_SUFFIX: "_test_only"
```

Comment:
You have to use the non test only image that is generated when the PR of build images gets merged, can you replace the image versions?

Reply:
Done now that the image has been merged!